content/docs/command-reference/commit.md

```diff
@@ -1,7 +1,7 @@
 # commit
 
-Record changes to DVC-tracked files in the <abbr>project</abbr>, by updating
-[DVC-files](/doc/user-guide/dvc-files-and-directories) and saving
+Record changes to DVC-tracked files in the <abbr>project</abbr>, by updating the
+[stages](/doc/command-reference/run) (or `.dvc` files) and saving
```

Comment:
It doesn't change stages, can you check what it does and try again? 🙂 
And no need for parenthesis around "and .dvc files".

Reply:
I think `commit` only saves data to cache, it doesn't update `dvc.lock` or .`dvc` files. But the previous definition had the updating part so I didn't change it.

Reply:
Hey sorry, I was out a few days. I'm back, let me review this again.

Reply:
Continued this conversation in https://github.com/iterative/dvc.org/pull/1626#pullrequestreview-460910286.